scheduler/src/cook/kubernetes/api.clj

```diff
@@ -622,9 +621,11 @@
           (.setWorkingDir container sidecar-workdir)
           (.setPorts container [(.containerPort (V1ContainerPort.) (int port))])
 
-          (.setEnv container [(make-env "COOK_WORKDIR" workdir)])
+          (.setEnv container (conj main-env-vars
+                                   (make-env "COOK_SCHEDULER_REST_URL" (config/scheduler-rest-url))
+                                   (make-env "COOK_WORKDIR" workdir)))
 
-          (.setPort http-get-action (IntOrString. port))
+          (.setPort http-get-action (-> port int IntOrString.))
```

Comment:
Nitpick: why did this change? Can port be a double/float?

Reply:
We discussed this in an earlier iteration. This was just to eliminate a reflection warning.